src/metabase/lib/schema/expression.cljc

```diff
@@ -203,3 +203,21 @@
 ;;; the `:expressions` definition map as found as a top-level key in an MBQL stage
 (mr/def ::expressions
   [:sequential {:min 1} [:ref ::expression.definition]])
+
+(mr/def ::positive-integer
```

Comment:
I know it's a mouthful, but maybe `::positive-integer-or-integer-expression` instead? If I saw `::positive-integer` in a schema, I'd assume it only allowed positive integers.